### DISCUSSION TITLE
Subtitle extraction taking too much time to complete

### DISCUSSION THREAD

**Discussion by bagajohny** [YaoFANGUK/video-subtitle-extractor#140] [General]
My specs,

OS - Windows 10 Pro 21H2
CPU - intel i5-6300HQ
GPU - Nvidia 950M 
VSE - latest GPU version of the software

I downloaded the software from [THIS LINK.](https://github.com/YaoFANGUK/video-subtitle-extractor/releases/download/1.0.0/vse_windows_GPU.7z)

I was trying to extract hardcoded subtitles from a 30min 720p video. First I tried "fast" mode but it gave srt file with 0 bytes. So then after checking the github issue I tried with "accurate" mode. But in accurate mode the software is taking very long time to complete the process. It only completed 10% of progress bar in 1 hr. At this rate it will take 10 hr time for just 30min video. Is it really that slow?
Should I try CPU version of the software instead? Will CPU version be faster than GPU version?

**Comment by YaoFANGUK** [comment]
Thank you for your feedback. I'm trying to solve this problem. If you got 0 bytes srt file, you may try to extract again. If every time you got 0 bytes, please send the video to me.

**Reply by bagajohny** [reply]
Yes, I am getting 0 byte srt file every time. This is an issue with GPU version in "fast" mode. I tried the CPU version is "fast" mode & I got proper srt file. I have not tried "accurate" mode for either version as it is taking too much time for me. So I aborted the process.

How can I send the video file to you? Is there any option to send PM? I am new to github so please excuse me.

**Reply by YaoFANGUK** [reply]
google drive pls. Thank you very much

**Reply by bagajohny** [reply]
I have mega drive link. I hope its okay.

[File Link](https://mega.nz/file/RHRDmbYT#C9kwhGdPcn97TlvgJjW4BRl7YPn-8ac0G0U41HtgtJk=)

**Reply by bagajohny** [reply]
@YaoFANGUK Did you check the file?
